pycqed/analysis/measurement_analysis.py

```diff
@@ -7098,12 +7098,15 @@ def run_default_analysis(self, print_fit_results=False, analyze_ef=False,
             except (TypeError, KeyError, ValueError):
                 logging.warning('qb_name is None. Old parameter values will '
                                 'not be retrieved.')
-                label = 'f0={:.5f} GHz $\pm$ ({:.2f}) MHz ' \
-                        '\nkappa0={:.4f} MHz $\pm$ ({:.2f}) MHz'.format(
-                            self.fit_res.params['f0'].value * scale,
-                            self.fit_res.params['f0'].stderr / 1e6,
-                            self.fit_res.params['kappa'].value / 1e6,
-                            self.fit_res.params['kappa'].stderr / 1e6)
+                try: #Dirty fix, should already be fine in Develop
```

Comment:
This is not how it should be fixed and this should not be part of this PR. 

Could you add the proper fix after merging? The proper fix uses the function `format_value_string` that can be found in `pycqed/utilities/general` examples of its use can be found in for example `analysis_v2/timedomain_analysis`

Reply:
Will fix after merging